{{cookiecutter.project_shortname}}/docker-services.yml

```diff
@@ -66,6 +67,12 @@ services:
     ports:
       - "9200:9200"
       - "9300:9300"
+  kibana:
+    image: docker.elastic.co/kibana/kibana:6.2.2
```

Comment:
This should be the `docker.elastic.co/kibana/kibana-oss:6.2.3` flavor: https://www.elastic.co/guide/en/kibana/current/_pulling_the_image.html